packages/wallet-connect-sdk-wallet-core/src/sdk.ts

```diff
@@ -214,11 +217,11 @@ export class WcWalletSDK {
       if (wccv > COMPATIBILITY_VERSION) throw new Error('Incompatible WCCV. Update your wallet to use new features.')
     }
 
-    const { topic } = await this.signClient.pair({
+    await this.web3wallet?.pair({
       uri,
     })
 
-    wccv && this.wccvs.set(topic, wccv)
+    //wccv && this.wccvs.set(topic, wccv)
```

Comment:
you forgot a commented code. I don't know is the wccvs is needed by Neon Wallet (please check), but if you really need to remove this line, also remove the wccvs property.